Index: storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config b/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config	
@@ -0,0 +1,2 @@
+%% This file is auto-generated! Edit at your own risk!
+{segment_entry_count, 2048}.
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config b/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config	
@@ -0,0 +1,2 @@
+%% This file is auto-generated! Edit at your own risk!
+{segment_entry_count, 2048}.
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name b/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name	
@@ -0,0 +1,2 @@
+VHOST: /
+QUEUE: downloadImages
Index: storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost b/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost	
@@ -0,0 +1,1 @@
+/
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name b/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name	
@@ -0,0 +1,2 @@
+VHOST: /
+QUEUE: amq.gen-8dwTRFAECiujHQ9qxACMUg
Index: storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config b/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@8aa26f2a151e/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config	
@@ -0,0 +1,2 @@
+%% This file is auto-generated! Edit at your own risk!
+{segment_entry_count, 2048}.
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config b/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@39fb788e3119/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.config	
@@ -0,0 +1,2 @@
+%% This file is auto-generated! Edit at your own risk!
+{segment_entry_count, 2048}.
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name b/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@165d2b3eaf24/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/Z2A7U61SCW3BW8T2WMUIDKY6/.queue_name	
@@ -0,0 +1,2 @@
+VHOST: /
+QUEUE: downloadImages
Index: storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost b/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@e6bd725364b1/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/.vhost	
@@ -0,0 +1,1 @@
+/
\ No newline at end of file
Index: storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name
===================================================================
diff --git a/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name b/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name
new file mode 100644
--- /dev/null	
+++ b/storage/docker/rabbitmq/data/rabbit@41504355b678/msg_stores/vhosts/628WB79CIFDYO9LJI6DKMI09L/queues/9YZT4UD0EES5RHH2P7F785ZNZ/.queue_name	
@@ -0,0 +1,2 @@
+VHOST: /
+QUEUE: amq.gen-8dwTRFAECiujHQ9qxACMUg
